app/models/comment.rb

```diff
@@ -1,4 +1,10 @@
 class Comment < ApplicationRecord
-  belongs_to :user
+  after_save :update_commentscounter
+
+  belongs_to :author, class_name: 'User'
   belongs_to :post
+
+  def update_commentscounter
+    post.increment!(:commentscounter)
+  end
```

Comment:
- [OPTIONAL]  Same here As requested by the previous reviewer, make sure the `update_commentscounter` is `private` 👍🏻 

- Note; Since this is the only think you're missing, I'm approving this PR but I strongly recommand you address this 💯 👍🏻

Reply:
done